commands/analytics.go

```diff
@@ -0,0 +1,24 @@
+package commands
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/section/sectionctl/analytics"
+)
+
+// AnalyticsCmd handles recording analytics events
+type AnalyticsCmd struct {
```

Comment:
Does this, and should this, command appear in `--help` output and tab completion?

Reply:
It does not (because of the [`hidden` parameter](https://github.com/section/sectionctl/pull/57/files#diff-ad4bfb563a6f3de9223c573150a0a0104111969f3055eace937c1226ad3395c7R27)). 

I don't believe it should show in the `--help` output, because it's an internal function of `sectionctl` that users shouldn't use directly. 

I just checked, and indeed it does show up in tab completion 🤦  I will have to work out a way to exclude it from tab completion.

Reply:
This required an upstream change, which can be found at WillAbides/kongplete#9.

Reply:
As of b214dc7, we're using a fork of kongplete that includes the `hidden` fixes. 

Once the PR is merged upstream, I'll change us back.